engines/watchmaker/console.cpp

```diff
@@ -0,0 +1,29 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
If you do not plan adding a debug console, it is better to remove this empty file